docs/06-artisan-commands.md

```diff
@@ -38,31 +38,45 @@ class InstallBlogPackage extends Command
         $this->info('Installing BlogPackage...');
 
         $this->info('Publishing configuration...');
-
-        //check if file exists
-        if(!File::exists(config_path('blogpackage.php'))) {
-            $this->publish_config();
+        
+        if ($this->configExists('blogpackage.php') === false) {
```

Comment:
I'm not sure, that we need strict compare between guaranteed boolean and false

Reply:
I confirm your changes. Sorry for bad English.

Reply:
I think you're right. I've updated it in my recent commit.